CRM/Aeapi/Submission.php

```diff
@@ -108,16 +108,12 @@ public static function getGroupIdByName($group_name) {
   // Check if contact already exists
   public static function isNew($contact_type, $contact_data) {
     $contact_data['contact_type'] = $contact_type;
-    $contact = civicrm_api3('Contact', 'get', $contact_data);
     try {
-      if(!empty($contact['id'])) {
-        return false;
-      } else {
-        return true;
-      }
+      civicrm_api3('Contact', 'getsingle', $contact_data);
+      return TRUE;
```

Comment:
Since the function is called "isNew", when getsingle does find someone, this function should return FALSE, right?

Reply:
You're abolutely right, I already fixed that in the second commit.